### PR TITLE
some minimization

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**
Make identifiers shorter and remove some data we collect by default 
[#73](https://github.com/Microsoft/ApplicationInsights-JS/issues/73)

### CODE REVIEW

File: JavaScript/JavaScriptSDK/Util.ts [hunk 1]
```diff
@@ -227,19 +227,18 @@ module Microsoft.ApplicationInsights {
...
-            var hexValues = ["0", "1", "2", "3", "4", "5", "6", "7", "8", "9", "A", "B", "C", "D", "E", "F"];
-
-            // c.f. rfc4122 (UUID version 4 = xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx)
-            var oct = "", tmp;
-            for (var a = 0; a < 4; a++) {
-                tmp = (4294967296 * Math.random()) | 0;
-                oct += hexValues[tmp & 0xF] + hexValues[tmp >> 4 & 0xF] + hexValues[tmp >> 8 & 0xF] + hexValues[tmp >> 12 & 0xF] + hexValues[tmp >> 16 & 0xF] + hexValues[tmp >> 20 & 0xF] + hexValues[tmp >> 24 & 0xF] + hexValues[tmp >> 28 & 0xF];
-            }
+        public static newId(): string {
+            var base64chars = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/';
```
Comment (MaxShehovtsov) [inline JavaScript/JavaScriptSDK/Util.ts]: Is it still gonna be a valid GUID?

File: JavaScript/JavaScriptSDK/Context/Session.ts [hunk 2]
```diff
@@ -16,11 +16,6 @@ module Microsoft.ApplicationInsights.Context {
         public id: string;
 
         /**
-         * The true if this is the first session
-         */
-        public isFirst: boolean;
```
Comment (MaxShehovtsov) [inline JavaScript/JavaScriptSDK/Context/Session.ts]: I thought that our UI in Settings->Usage->Users->New users chart is based on isFirst flag. Can you double check pls

File: JavaScript/JavaScriptSDK/Util.ts (hunk 1 above)
Comment (SergeyKanzhelev) [inline JavaScript/JavaScriptSDK/Util.ts]: Nope. Do you know any places we are using it as a GUID?

File: JavaScript/JavaScriptSDK/Context/Session.ts (hunk 2 above)
Comment (SergeyKanzhelev) [inline JavaScript/JavaScriptSDK/Context/Session.ts]: Yes, you are correct. We are still using it to detect "new users" vs. "returning users". I really wonder why it is not a field on SessionState event. But nevertheless I returned it back

File: JavaScript/JavaScriptSDK/Util.ts (hunk 1 above)
Comment (MaxShehovtsov) [inline JavaScript/JavaScriptSDK/Util.ts]: Does server side SDK (including Java and Node) treat is as a string? If yes then you should be fine.

Comment (SergeyKanzhelev) [inline JavaScript/JavaScriptSDK/Util.ts]: Validated. It is string everywhere

**Comment by SergeyKanzhelev** [comment]
@MaxShehovtsov  is it looking ok now? Note, I removed `accountAcquisitionDate` if compare to the Friday and returned isFirst for the session context back

**Comment by MaxShehovtsov** [comment]
Looks good. Merging. Thanks!

**Pull request merged**